### ISSUE TITLE
Running "abi-types-generator" command fails on ubuntu

### ISSUE THREAD

**Issue by afmfe-iul** [joshstevens19/ethereum-abi-types-generator#17]
Hey there! First of all thanks a lot for this package.

On to the issue! I've tried to run `abi-types-generator` on an existing project and it failed, I was not sure if the problem was on my end so I cloned the repository and tried to run the examples and I got the same error:
```
> yarn web3-example
yarn run v1.22.5
$ abi-types-generator './abi-examples/fake-contract-abi.json' --output='./web3/fake-contract-example/generated-typings' --name=fake-contract
/usr/bin/env: ‘node\r’: No such file or directory
error Command failed with exit code 127.
```

After a quick google search of that error message, and since I'm working on Ubuntu, I tried [this solution](https://stackoverflow.com/a/40414061) and it fixed my problem.

Let me know if this makes sense and if there's anything I can help with!

**Comment by joshstevens19** [comment]
Hey thanks a lot for raising this issue. Yeah this package was written on windows so that error makes sense. Saying that npm fixed this problem on version 5.4.0 but yarn I’m afraid have not. Can you confirm if you use npm to run the script it works? I can have a look at putting that /r fix in the bin file as well and then test it on Ubuntu and windows to make sure it’s not broken anything.  

I think you can still use the package right? Or is this a blocker for you?

**Comment by afmfe-iul** [comment]
Hey @joshstevens19 thanks for the quick reply!

I've double checked and if I run the script with npm it still fails because we install our packages with yarn. Just for testing, I tried removing the project node modules and installing them with npm and running the script with npm, and this problem I'm reporting was fixed. However swapping to npm is not an option for us currently.

We have the above hack to fix the issue so we can still use the package, but a fix at the root would be awesome! :)

**Comment by joshstevens19** [comment]
Have not forgot about this i promise! sorry been very busy its on my TODO list

**Comment by joshstevens19** [comment]
Hey so now the compiler will make the new line be an LF which seems to work fine on windows as well. Cant check on Linux so if you can confirm this has fixed it that be great > version 1.1.8 :+1:

**Comment by afmfe-iul** [comment]
@joshstevens19 nicely done! Can confirm that it works on version 1.1.8. Keep up the good work :)
